dotCMS/src/integration-test/java/com/dotcms/tika/TikaUtilsTest.java

```diff
@@ -0,0 +1,105 @@
+package com.dotcms.tika;
+
+import com.dotmarketing.exception.DotDataException;
+import java.util.HashMap;
+import java.util.HashSet;
+import java.util.Map;
+import java.util.Set;
+import org.junit.Assert;
+import org.junit.Test;
+
+/**
+ * @author nollymar
+ */
+public class TikaUtilsTest{
+
+    @Test
+    public void testGetConfiguredMetadataFields() throws DotDataException {
+        final TikaUtils tikaUtils = new TikaUtils();
+        final Set<String>  fields = tikaUtils.getConfiguredMetadataFields();
+
+        Assert.assertNotNull(fields);
+        Assert.assertTrue(!fields.isEmpty());
+    }
+
+    @Test
+    public void test_FilterMetadataFields_WhenMapEmpty_ReturnsEmptyMap() throws DotDataException {
```

Comment:
I think dataProviders could have been used for these cases? Just to take into consideration